instance/models/instance.py

```diff
@@ -586,4 +697,41 @@ def provision(self):
 
         return (server, ansible_log)
 
+    def provision_external_databases(self):
+        """
+        Provision external databases for this instance
+        """
+        self.provision_external_mysql()
+        self.provision_external_mongo()
+
+    def provision_external_mysql(self):
+        """
+        Provision external mysql database for this instance
+        """
+        if settings.INSTANCE_MYSQL_URL_OBJ:
+            connection = mysql.connect(
+                host=settings.INSTANCE_MYSQL_URL_OBJ.hostname,
+                user=settings.INSTANCE_MYSQL_URL_OBJ.username,
+                passwd=settings.INSTANCE_MYSQL_URL_OBJ.password or '',
+                port=settings.INSTANCE_MYSQL_URL_OBJ.port or 3306,
+            )
+            cursor = connection.cursor()
+
+            # We can't use the database name in a parameterized query, the
+            # driver doesn't escape it properly. Se we escape it here instead
+            database_name = connection.escape_string(self.database_name).decode()
```

Comment:
@omarkhan Is there a way to not need this escaping, to be able to use the `database_name` directly everywhere? For example by only chosing characters that don't need escaping when chosing the base database name, initially?

Also, since there are several databases, it's not obvious from the attribute name that `self.database_name` is the name of the MySQL database. The way you build DB names out of a base DB name by appending suffixes is a good approach imho, but to be consistent the MySQL DB name should also be built the same way - ie having a `self.mysql_database_name` attribute.